build-tools/create-packs/Directory.Build.targets

```diff
@@ -165,6 +165,49 @@
     <RemoveDir Directories="@(_PackFoldersToDelete)" />
   </Target>
 
+  <Target Name="ExtractMauiWorkloadPacks" >
```

Comment:
There is also an `InstallMaui` target:

https://github.com/xamarin/xamarin-android/blob/3653ec3dfaf090d6f3aec7cd193056a9d10197d7/build-tools/scripts/DotNet.targets#L45

This one, I've used to install `maui-android` locally from a build on the `dotnet8` feed.  For example:
```
dotnet build Xamarin.Android.sln -t:InstallMaui -p:MauiVersion=8.0.0-preview.6.8686 -p:MauiVersionBand=8.0.100-preview.6
```

Do you think we still need both of these targets?

Reply:
Ahh good point, I've tried to merge my changes into `InstallMaui` to allow that target to run against locally built packages.